docs/en/Configuration-Settings.md

```diff
@@ -8,50 +8,77 @@ priority: 1
+in commandline. For example, the following Alluxio shell command sets the write type to
+`CACHE_THROUGH` when copying files to Alluxio:
 
-### Application settings
+```bash
+$ bin/alluxio fs -Dalluxio.user.file.writetype.default=CACHE_THROUGH copyFromLocal README.md /README.md
+```
+
+## Spark Jobs
+
+Spark users can pass JVM system properties to Spark jobs by adding `"-Dproperty=value"` to
+`spark.executor.extraJavaOptions` for Spark executors and `spark.driver.extraJavaOptions` for
+Spark drivers. For example, the following Spark job sets write type to `CACHE_THROUGH` when
+writing to Alluxio:
 
-Alluxio shell users can use `-Dkey=property` to specify an Alluxio configuration value in commandline. For example,
+```bash
+$ spark-submit \
+--conf 'spark.driver.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+--conf 'spark.executor.extraJavaOptions=-Dalluxio.user.file.writetype.default=CACHE_THROUGH' \
+...
+```
 
-{% include Configuration-Settings/specify-conf.md %}
+## Hadoop MapReduce Jobs
 
-Spark users can add `"-Dkey=property"` to `${SPARK_DAEMON_JAVA_OPTS}` in `conf/spark-env.sh`, or add it to
-`spark.executor.extraJavaOptions` (for Spark executors) and `spark.driver.extraJavaOptions` (for Spark drivers).
+Hadoop MapReduce users can add `"-Dproperty=value"` in `hadoop jar` or `yarn jar` command and the
+properties will be propagated to all the tasks of this job.  For example, the following MapReduce
+job of wordcount sets write type to `CACHE_THROUGH` when writing to Alluxio:
+
+```bash
+$ bin/hadoop jar libexec/share/hadoop/mapreduce/hadoop-mapreduce-examples-2.7.3.jar wordcount \
+-Dalluxio.user.file.writetype.default=CACHE_THROUGH \
+-libjars {{site.ALLUXIO_CLIENT_JAR_PATH}} \
+<INPUT FILES> <OUTPUT DIRECTORY>
+```
 
-Hadoop MapReduce users can set `"-Dkey=property"` in `hadoop jar` command-lines to pass it down to Alluxio:
+# Configure Alluxio Cluster
 
-{% include Configuration-Settings/hadoop-specify-conf.md %}
+## Use Site-Property Files (Recommended)
 
-Note that, setting Alluxio configuration in this way is application specific and required for each job or command.
+Alluxio admin can create and customize Java property file `alluxio-site.properties` to
+configuration an Alluxio cluster. If this file does not exist, it can be simply created from a
+template file:
 
-### Environment variables
+```bash
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+```
 
-> When you want to [start Alluxio server processes](Running-Alluxio-Locally.html), or [use Alluxio command line interfaces](Command-Line-Interface.html) with your specific configuration tuning, it is often fast and easy to set environment variables to customize basic Alluxio configuration. However, these environment variables will not affect application processes like Spark or MapReduce that use Alluxio as a client.
+On startup, Alluxio runtime searches `alluxio-site.properties` in `${HOME}/.alluxio/`,
```

Comment:
instead of saying how things work, we should say when each location should be used.